kalite/topic_tools/__init__.py

```diff
@@ -151,7 +152,7 @@ def get_exercise_cache(force=False, language=settings.LANGUAGE_CODE):
                 items = []
                 for item in exercise.get("all_assessment_items","[]"):
                     item = json.loads(item)
-                    if assessmentitems.get(item.get("id")):
+                    if main_models.AssessmentItem.objects.filter(id=item.get("id")).exists():
```

Comment:
Sure, it was my hope that using `exists` might be faster, but I think it typically performs the same query as actually getting the row anyway.